src/Symfony/Bundle/FrameworkBundle/Routing/RouteLoaderInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Routing;
+
+use Symfony\Component\Config\Loader\Loader;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Defines a class that is able to load and return a RouteCollection.
+ *
+ * @author Ryan Weaver <ryan@knpuniversity.com>
+ */
+interface RouteLoaderInterface
+{
+    /**
+     * @param Loader $loader
+     *
+     * @return RouteCollection
+     */
+    public function getRouteCollection(Loader $loader);
```

Comment:
My problem is that the `Loader` has `import` on it, which is really what the user will want. The `LoaderInterface` doesn't have this. What do you think about creating a new interface for loaders with `import` to avoid type-hinting the implementation?